test/exportapi.test.js

```diff
@@ -0,0 +1,180 @@
+/*global describe, beforeEach, it */
+'use strict';
+var path = require('path');
+var helpers = require('yeoman-generator').test;
+var SANDBOX =  path.resolve(__dirname, 'sandbox');
+var fs = require('fs');
+var expect = require('chai').expect;
+var wsModels = require('loopback-workspace').models;
+var common = require('./common');
+var yaml = require('js-yaml');
+
+describe('loopback:model generator', function() {
```

Comment:
I think this should be called `loopback:export-api-def` instead.

Also please rename the test file to `export-api-def.test.js` so that it matches the generator name.